qld-gov-au/qgds-bootstrap5
# [QOLOE-419] Accessibility: Feedback input fields lost focus state after user inputs

[QOLOE-419] Accessibility: Feedback input fields lost focus state after user input.

## The bug
The outline styling for `focus` state is missing after user completed the form fields. FormIO adds 'form-modified' class whenever a field has been completed. 

## The fix
It applied the same outline styling on `focus` state regardless whether the form field has been completed.
An additional fix: background colour for the textarea on focus state should be white.

## To test
- Go to Feedback form on this PR's Storybook
- Tab through each field. An outline should appear on the field when focused via tab. 
- Complete the fields - without submitting the form.
- Tab back to the previously completed field
- Bug: Field does not have outline when focused via tab
- Fix: Field has outline when focused via tab


[QOLOE-419]: https://ssq-qol.atlassian.net/browse/QOLOE-419?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
